lib/ostiary/policy.rb

```diff
@@ -1,18 +1,20 @@
 module Ostiary
   class Policy
-    attr_accessor :name, :rules
+    attr_reader :name, :method, :actions
```

Comment:
I was questioning why you would rename :rules to :actions, but these are the controller actions of course

Reply:
yeah it confused me quite a bit, especially since a bit was renaming in c91224c04f1439d148f6b0c16024081164cf7c08